frontend/src/components/TournamentManager.js

```diff
@@ -39,13 +39,14 @@ import CreateTournamentForm from "./tournament/CreateTournamentForm";
 import UpdateSpiritScoreForm from "./tournament/UpdateSpiritScoreForm";
 import { initFlowbite } from "flowbite";
 import clsx from "clsx";
+import ReorderTeams from "./tournament/ReorderTeams";
 
 const TournamentManager = () => {
   const queryClient = useQueryClient();
   const [store] = useStore();
   const [selectedTournament, setSelectedTournament] = createSignal();
   const [selectedTournamentID, setSelectedTournamentID] = createSignal(0);
-  const [tournamentSeeding, setTournamentSeeding] = createSignal([]);
+  const [teamSeeding, setTeamSeeding] = createSignal([]);
```

Comment:
Is there a reason behind this name change? Just wanted to understand why you find this a better name.

Reply:
`tournamentSeeding` was being initialised to a dictionary previously. https://github.com/SibiAkkash/hub/blob/8f71f8edb2aba6e8af39d10883aa1fce7f20e9e8/frontend/src/components/TournamentManager.js#L86 and https://github.com/SibiAkkash/hub/blob/8f71f8edb2aba6e8af39d10883aa1fce7f20e9e8/frontend/src/components/TournamentManager.js#L370

Reply:
Ah, makes sense! Just `teams` or `orderedTeams` might perhaps be better names?

Reply:
yes, i'll rename it to teams